osu.Game/Graphics/UserInterface/ShearedSearchTextBox.cs

```diff
@@ -0,0 +1,192 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Input;
+using osu.Framework.Input.Events;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays;
+using osuTK;
+using osuTK.Graphics;
+using osuTK.Input;
+
+namespace osu.Game.Graphics.UserInterface
```

Comment:
Was gonna mention that this should probably be placed in `UserInterfaceV2`, but the sheared overlay implementations are already in `UserInterface`...

Reply:
Doesn't matter at this point I think. Most of the V2 stuff is going to get redesigned again anyhow.